### ISSUE TITLE
Update our miniconda download in the continuous integration

### ISSUE THREAD

**Issue by GaelVaroquaux** [nilearn/nilearn#1672]
We are downloading an old version of miniconda in the continuous integration scripts, as mentioned in https://github.com/conda/conda/issues/7481#issuecomment-402164747

The proper link is of the form:
```
wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
```

**Comment by KamalakerDadi** [comment]
+1

**Comment by kchawla-pi** [comment]
This might be fixed in PR #1691 .

**Comment by GaelVaroquaux** [comment]
Fixed in #1691
